src/lwip/app_ethernet.c

```diff
@@ -110,7 +175,10 @@ void dhcp_task(void *pv_params) {
         BSP_LED_Off(LED2);
         BSP_LED_Off(LED3);
 
-        dhcp_start(netif);
+        err_t dhcp_init_result = dhcp_start(netif);
+        if (dhcp_init_result != 0) {
+          printf("Something went when initiating DHCP discover");
```

Comment:
This `printf` isn't actually doing anything since stdout/stderr is not linked to any backend. you can probably remove this whole if statement.

Reply:
Ya just a testing if conditional for now cause I was just checking if DHCP discovers were not sent. Should we have re-attempts of DHCP Start initiated if this fails?